ddl/ddl_api.go

```diff
@@ -2187,7 +2187,7 @@ func (d *ddl) RenameIndex(ctx sessionctx.Context, ident ast.Ident, spec *ast.Alt
 }
 
 // DropTable will proceed even if some table in the list does not exists.
-func (d *ddl) DropTable(ctx sessionctx.Context, ti ast.Ident) (err error) {
+func (d *ddl) DropTable(ctx sessionctx.Context, ti ast.Ident, isView bool) (err error) {
```

Comment:
It seems that we do not need `isView`.
Check tb.Meta().View == nil in this function is enough？